app/controllers/api/services_controller.rb

```diff
@@ -159,4 +154,8 @@ def authorize_action
   def can_create?
     can? :create, Service
   end
+
+  def presenter
+    @presenter ||= Api::ServicesIndexPresenter.new(current_user: current_user, params: params)
+  end
```

Comment:
Where is this method used?

Reply:
It's used in the view: https://github.com/3scale/porta/blob/2cb37251a80941ba29d2a45fecb116431f8a96c0/app/views/api/services/index.html.slim#L1